### PR TITLE
docs: add telemetry section

### PR DESCRIPTION
**Pull request by stefanicjuraj**
This is a draft pull request containing a telemetry section addition. The telemetry section is ported from OSS documentation - should there be any differences between the OSS and Enterprise telemetry sections?

Fixes https://github.com/daytonaio/enterprise-docs/issues/102
cc @fabjanvucina

### CODE REVIEW

**Comment by fabjanvucina** [comment]
@stefanicjuraj The telemetry data collection feature is still pending in the enterprise product. For the time being, the telemetry section in the admin dashboard is only for giving/revoking consent.

File: src/content/docs/misc/telemetry.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+description: Information about telemetry data gathered by Daytona.
+sidebar:
+  label: Telemetry
+---
+
+All telemetry data is **completely anonymous** and **does not** include any personally identifiable information (PII) or user secrets. Participation is optional and you may opt-out at any time.
+
+## The Goal
+
+The goal of gathering telemetry data in the CLI is exclusively for the purpose of improving the Daytona ecosystem.
```
Comment (ivan-burazin) [inline src/content/docs/misc/telemetry.mdx:12]: >The goal of gathering telemetry data in the CLI is
We don't collect in the CLI here but rather in generally in daytona

**Review by ivan-burazin — CHANGES_REQUESTED** [review]
All mentions of CLI should be removed as data will be collected centeraly, which can include; cli, plugins, gui, ect.

You can confirm with @gdraganic about details

**Comment by ivan-burazin** [comment]
adding @gdraganic to verify this.

**Comment by fabjanvucina** [comment]
Commenting again for visibility. 

In the Enterprise product, telemetry data collection is a feature that hasn't yet reached the development stage. For the time being, the telemetry section in the admin dashboard is only for giving/revoking consent for when we start collecting the data.

@ivan-burazin Knowing this, how do you want to proceed with this PR?

**Comment by ivan-burazin** [comment]
> Commenting again for visibility.
> 
> In the Enterprise product, telemetry data collection is a feature that hasn't yet reached the development stage. For the time being, the telemetry section in the admin dashboard is only for giving/revoking consent for when we start collecting the data.
> 
> @ivan-burazin Knowing this, how do you want to proceed with this PR?

We should say just that. "telemetry data collection is a feature that hasn't yet reached the development stage. For the time being, the telemetry section in the admin dashboard is only for giving/revoking consent for when we start collecting the data."

**Comment by ivan-burazin** [comment]
@stefanicjuraj can you please update so we can merge? thanks

**Review by fabjanvucina — APPROVED** [review]

**Pull request merged**